esp/src/eclwatch/stub.js

```diff
@@ -35,13 +35,13 @@ define([
         if (modernMode === String(true) && hpccWidget !== "IFrameWidget") {
             switch (hpccWidget) {
                 case "WUDetailsWidget":
-                    window.location.replace(`/esp/files/index.html#/workunits/${params.Wuid}`);
+                    window.location.replace(`/#/workunits/${params.Wuid}`);
                     break;
                 case "GraphsWUWidget":
-                    window.location.replace(`/esp/files/index.html#/workunits/${params.Wuid}/metrics`);
+                    window.location.replace(`/#/workunits/${params.Wuid}/metrics`);
                     break;
                 default:
-                    window.location.replace("/esp/files/index.html");
+                    window.location.replace("/");
```

Comment:
Sanity checks:
* Did you test this works when switching between V9 and dojo pages?
* Does this code match what we have in the 9.2.x branch now?

Reply:
I did test that this works as expected when switching versions. But no, it doesn't match what's currently in 9.2. I thought the other PR (#17843) targeting 8.12.x should be merged up into 9.0 and so on. Also, regarding back porting, I believe Stu had pointed out in an ECL Watch meeting that the folks who'd encountered this had only fairly recently moved to 8.10 and they could be there for a while yet.